src/main/java/com/pablissimo/sonar/TsLintSensor.java

```diff
@@ -96,9 +96,28 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
 
             String lowerFilePath = filePath.toLowerCase();
 
+
+
             if (!fileMap.containsKey(lowerFilePath)) {
                 LOG.warn("TsLint reported issues against a file that wasn't sent to it - will be ignored: " + lowerFilePath);
-                continue;
+
```

Comment:
@Pablissimo 
 I think this issues not  resolved in the RC: 25a8523
Because TSLint have a bug https://github.com/palantir/tslint/issues/1794

```
c:/Users/tim/Documents/Project/src/utils/color/color-convert.ts(31,1): warning maxClassesPerFile: A maximum of 1 class per file is allowed
c:/Users/tim/Documents/Project/src/devices/dal/device-configure.ts(66,13): warning typeofCompare: typeof must be compared to correct value
src/graph/components/axis/x-axis.ts(42,1): warning maxClassesPerFile: A maximum of 1 class per file is allowed
src/graph/components/axis/y-axis.ts(34,1): warning maxClassesPerFile: A maximum of 1 class per file is allowed
```